src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,10 @@
+.stars {
+  display: flex;
+  height: 15px;
+
+  &__star {
+    width: 15px;
+    margin-right: 4px;
+    background: url("../images/star.svg") no-repeat center;
+  }
+}
```

Comment:
Misha mentioned that we have to make block stars, so it's worth adding to this block everything related to stars